lib/utils/helpers.js

```diff
@@ -131,6 +127,19 @@ const searchResult = (foundArticles, res) => {
   return successResponse(res, ARTICLE_FOUND_MESSAGE, OK_CODE, data);
 };
 
+/**
+ *
+ * @returns {string} adminStatus
+ * @param {*} id
+ */
+async function getUserRole(id) {
```

Comment:
Why do we have this function here?

Reply:
it is a helper function that returns a list of users roles